src/app/components/cds/charts/ListWidgetItem.js

```diff
@@ -16,7 +16,7 @@ const ListWidgetItem = ({
         :
         <div className={styles.listWidgetItemText}>{itemText}</div>
     }
-    <div className={styles.listWidgetItemValue}>{itemValue}</div>
+    <div className={styles.listWidgetItemValue}>{itemValue || '-'}</div>
```

Comment:
Suggestion: I think you could also change the default prop `itemValue` to be `null` so the fallback to `-` is in a single place.